compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -156,12 +151,43 @@ internal actual class ComposeWindow : UIViewController {
             if (bottomIndent < keyboardHeight) {
                 keyboardOverlapHeightState.value = (keyboardHeight - bottomIndent).toFloat()
             }
+
+            if (configuration.onFocusBehavior == OnFocusBehavior.FocusableAboveKeyboard) {
+                val focused = layer.getActiveFocusRect()
+                if (focused != null) {
```

Comment:
It's a good candidate for extracting to separate function

Reply:
This code is used in one place. I think we can leave it here.

Reply:
Done